system-test/fixtures/sample/src/index.js

```diff
@@ -20,8 +20,9 @@
 const containeranalysis = require('@google-cloud/containeranalysis');
 
 function main() {
-  const containerAnalysisClient = new containeranalysis.ContainerAnalysisClient();
+  const containerAnalysisV1Beta1Client = new containeranalysis.ContainerAnalysisV1Beta1Client();
```

Comment:
This is unexpected...

Reply:
The problem with this API is that clients have different names in `v1` and `v1beta1`. I think I will just change the order of versions to change this back.